app/src/test/java/org/oppia/android/app/testing/administratorcontrols/AdministratorControlsFragmentTest.kt

```diff
@@ -172,6 +173,8 @@ class AdministratorControlsFragmentTest {
   }
 
   private fun createAdministratorControlsActivityIntent(profileId: Int): Intent {
+    val profileId = ProfileId.newBuilder().setInternalId(profileId).build()
+
```

Comment:
Please differentiate between profileId and internalprofileId to avoid ambiguity.

Reply:
Okay.